docs/contract-interface/serialization-interface.md

```diff
@@ -35,6 +35,26 @@ pub fn sum_borsh(#[serializer(borsh)] a: u32, #[serializer(borsh)] b: u32) -> u3
 
 Where the `result_serializer(borsh)` annotation will override the default result serialization protocol from JSON to borsh and the `serializer(borsh)` annotations will override the parameter serialization.
 
+#### Example
+
+A simple demonstration of getting a [Borsh-serialized](https://borsh.io), base64-encoded value from a unit test:
+
+```rust reference
+https://github.com/mikedotexe/rust-status-message/blob/03781079d1716584d114bec294de00f5336cd20d/src/lib.rs#L106-L114
+```
+
+The following snippet shows a simple function that takes this value from a frontend or CLI. Note: this method doesn't have a return value, so the `#[result_serializer(borsh)]` isn't needed.
+
+```rust reference
+https://github.com/near/rainbow-bridge-rs/blob/b9096189af828d3684c9124dbd0c0072a35cd9f7/mintable-fungible-token/src/lib.rs#L399-L401
```

Comment:
Are you sure this is what you meant to link? https://github.com/near/rainbow-bridge-rs/blob/b9096189af828d3684c9124dbd0c0072a35cd9f7/mintable-fungible-token/src/lib.rs#L399-L401

Reply:
I think I made some silly approaches here, pushed a change that should be much better, thanks for the comments